talk/tools/sanitizers.tex

```diff
@@ -210,3 +210,53 @@
   \end{alertblock}
 
 \end{frame}
+
+\begin{frame}[fragile]
+  \frametitle{Thread sanitizer (tsan)}
+  \begin{block}{TSan}
+    \begin{itemize}
+      \item Thread sanitizer detects many data races in MT programs
+      \item Recompile your program with e.g.\ \mintinline{shell}{clang++ -fsanitize=thread -g -O1 datarace.cpp}
+    \end{itemize}
+  \end{block}
+
+  \footnotesize
+  \begin{verbatim}
+% ./a.out
+WARNING: ThreadSanitizer: data race (pid=19219)
+Write of size 4 at 0x7fcf47b21bc0 by thread T1:
+  #0 Thread1 datarace.c:4 (exe+0x00000000a360)
+
+Previous write of size 4 at 0x7fcf47b21bc0 by main thread:
+  #0 main datarace.c:10 (exe+0x00000000a3b4)
+
+Thread T1 (running) created at:
+  #0 pthread_create tsan_interceptors.cc:705 (exe+0x00000000c790)
+  #1 main datarace.c:9 (exe+0x00000000a3a4)
+  \end{verbatim}
+
+  \begin{block}{}
+    \scriptsize
+    \url{https://github.com/google/sanitizers/wiki/ThreadSanitizerCppManual}
+  \end{block}
+\end{frame}
+
+\begin{frame}[fragile]
+  \frametitle{Undefined Behaviour Sanitizer (UBSan)}
```

Comment:
```suggestion
  \frametitle{Undefined Behaviour Sanitizer (ubsan)}
```
You also spelled it lowercase on the tsan slide.

Reply:
They are actually spelled uppercase by the authors, so I'll do the inverse of what you suggest. 🙂